crates/qt-gui/qml/main.qml

```diff
@@ -0,0 +1,95 @@
+import QtCore
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Window 2.12
+import QtQuick.Dialogs
+import QtQuick.Layouts
+
+// This must match the uri and version
+// specified in the qml_module in the build.rs script.
+import com.kdab.cxx_qt.demo 1.0
+
+ApplicationWindow {
+    height: 480
+    title: qsTr("Rustagram Image Converter")
+    visible: true
+    width: 640
+
+    ColumnLayout {
+        anchors.fill: parent
+        anchors.margins: 10
+        spacing: 10
+
+        RowLayout {
+          Button {
```

Comment:
inconsistent spacing could be fixed :-) elsewhere in this QML file it is 4 spaces